x-pack/plugins/security_solution/public/management/cypress/e2e/endpoint/response_console.cy.ts

```diff
@@ -81,6 +81,26 @@ describe('Response console', () => {
       waitForEndpointListPageToBeLoaded(endpointHostname);
       checkEndpointListForOnlyUnIsolatedHosts();
     });
+
+    it('should fail if data tampered', () => {
```

Comment:
is this test case specific to the `isolate` command? if not specific, I'd suggest to move it out of this test group, which is sort of a user journey where a test case may depend on the previous case, but this new test case looks mostly independent, testing an error case outside of the user journey

Reply:
great point, will move.